boilerplate/index.ts

```diff
@@ -546,7 +559,7 @@ export function createBoilerplate(sdk: CDP) {
                                 return _(bpMappings).differenceWith(rMappings, _.isEqual).isEmpty();
                             };
 
-                            if (remoteMappings.length < 1) {
+                            if (remoteMappings.length < 1 || !remoteMappings) {
```

Comment:
condition order doesn't make sense. if `!remoteMappings` then the first part of the condition will throw.